python/sbp/client/forwarder.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Mark Fine <mark@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+from threading import Thread
+
+class Forwarder(Thread):
```

Comment:
Can a Forwarder be composed with its reverse to duplex observations? It doesn't seem so. I want to be able to preserve a capability for reading/writing from the serial port and a network connection (TCP/HTTP, etc.).